config/jobs/kubernetes-sigs/kueue/kueue-periodic-s390x-ppc64le-main.yaml

```
@@ -1,10 +1,10 @@
 periodics:
   - interval: 12h
-    name: periodic-kueue-build-s390x-ppc64le-main
+    name: periodic-kueue-build-s390x-ppc64le-main-multiarch
```

```suggestion
    name: periodic-kueue-build-s390x-ppc64le-main
```

I don't think that this `multiarch` suffix is helpful.
Same comments on other places as well.